module/documents/actor/actor.mjs

```diff
@@ -1512,13 +1512,16 @@ export default class Actor5e extends Actor {
   async rollInitiativeDialog(rollOptions={}) {
     // Create and configure the Initiative roll
     const roll = this.getInitiativeRoll(rollOptions);
-    const choice = await roll.configureDialog({
-      defaultRollMode: game.settings.get("core", "rollMode"),
-      title: `${game.i18n.localize("DND5E.InitiativeRoll")}: ${this.name}`,
-      chooseModifier: false,
-      defaultAction: rollOptions.advantageMode ?? dnd5e.dice.D20Roll.ADV_MODE.NORMAL
-    });
-    if ( choice === null ) return; // Closed dialog
+
+    if (rollOptions.advantageMode === null) {
```

Comment:
This would introduce a difference between how initiative checks work compared to all other d20 rolls. Looking at how the `d20Roll` function works, and the `determineAdvantageMode` function it uses, it requires either a `fastForward` property or holding down one of the special keys (e.g. alt for advantage). This skips the dialog if the `advantageMode` is set rather than using an explicit `fastForward` or one of the key presses.

For the Advantage Reminder module, I'm looking at setting the `advantageMode` property and I think it would be a bad user experience to fast forward the roll in this case. And while not related to your PR, while I understand we can't use `d20Roll` in this case since we need to return an unrolled `Roll` instance, I think we should keep it as close to that for consistency sake for module/macro developers.

Reply:
Yup, just pushed a change; it's using the `determineAdvantageMode()` function and rather sets the `isFF` property instead.

Given the new changes, what do you suggest? I am not super comfortable in the code-base - I've just started - so I have no idea how the existing module/macro developers use things.